cypress/page/payment-step.page.ts

```diff
@@ -1,12 +1,19 @@
 class PaymentStepPage {
     private bankWireTag: string;
+    private cartNav: string;
 
     constructor() {
         this.bankWireTag = ".bankwire";
+        this.cartNav = "#cart_navigation > .button";
     }
 
     public payByBank() {
         cy.get(this.bankWireTag).click();
+        cy.get(this.cartNav).click();
+    }
+
+    public getConfirmationMessage() {
+        cy.get("#center_column > div.box > p > strong").should("have.text", "Your order on My Store is complete.");
```

Comment:
store the locator in a variable (I would call it confirmationMessageText, but you can choose another name), also, make this method to return the element with the title:
`return cy.get(this.confirmationMessageText);`

The assertion (`.should("have.text", "Your order on My Store is complete.");`) should go on the buy-tshirt.cy.ts file